db/migrate/20200310222258_add_columns_to_prosecution_case_defendant_offences.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddColumnsToProsecutionCaseDefendantOffences < ActiveRecord::Migration[6.0]
+  def change
+    add_column :prosecution_case_defendant_offences, :maat_reference, :string
+    add_column :prosecution_case_defendant_offences, :dummy_maat_reference, :boolean
+    add_column :prosecution_case_defendant_offences, :response_status, :integer
+    add_column :prosecution_case_defendant_offences, :response_body, :json
+    add_column :prosecution_case_defendant_offences, :user_id, :string
```

Comment:
This is an interesting one, at the moment, we authenticate with the UI via `client_credentials`, but I guess we can leave it to the UI to inform us of the requesting user.
In terms of the user format, should we probably agree with the UI on our their format before we add this one?

Reply:
Fair point. I'll remove it for now and add a ticket to add it in later when we've agreed how this will work